src/app/docs/icon/icon-docs.component.html

```diff
@@ -4,143 +4,170 @@
+          </p>
+        </li>
+      </ul>
+
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Icon styling"
+    >
+      <p class="sky-docs-text-block">
+        SKY UX uses simple, glyphic, mono-color icons to facilitate, guide, and inform users. Icons are designed in Blackbaud's brand personality, reflecting a human, visionary approach, and are built on the SKY UX's modern theme's visual language.
       </p>
 
       <sky-docs-design-guideline
-        heading="How to select icons"
+        heading="Color"
       >
-        <p>
-          To determine whether an icon is available for a new component, check the SKY UX icon dictionary below. Then make sure that your use case is consistent with how other components use the icon.
+        <p class="sky-docs-text-block">
+          Icons are always a solid, monochromatic color and must meet the <a href="https://www.w3.org/WAI/WCAG21/Techniques/general/G207.html" target="_blank">WCAG contrast ratio of 3:1</a>. An icon's color should be reflective of the component to which it is tied. For more information, view the SKY UX color guidelines for modern theme.
         </p>
-
-        <ul>
-          <li>
-            <p>
-              Do not use icons with established meanings for different purposes.
-            </p>
-          </li>
-          <li>
-            <p>
-              Be mindful of how icons are used in specific components such as <a href="https://developer.blackbaud.com/skyux/components/grid">grids</a>, <a href="https://developer.blackbaud.com/skyux/components/tiles">tiles</a>, <a href="https://developer.blackbaud.com/skyux/components/button">buttons</a>, and <a href="https://developer.blackbaud.com/skyux/components/modal">modals</a>
-            </p>
-          </li>
-        </ul>
-        <p>
-          Do not use icons with established meanings for different purposes.
+      </sky-docs-design-guideline>
+      <sky-docs-design-guideline
+        heading="Variations"
+      >
+        <p class="sky-docs-text-block">
+          SKY UX icons have two different style variations: outlined and filled. Use the outlined variation for all icons displayed within your experiences. The filled variation is only used to indicate state changes on a given component as described in its documentation.
         </p>
       </sky-docs-design-guideline>
+      <sky-docs-design-guideline
+      heading="Sizing"
+    >
+      <p class="sky-docs-text-block">
+        Icons usually inherit their size and color from parent elements, particularly when you use them inline. SKY UX icons are typically 16px to match standard body copy. Icons of varying size can be used depending on its parent's component specifications. 
+      </p>
+    </sky-docs-design-guideline>
     </sky-docs-demo-page-section>
 
     <sky-docs-demo-page-section
```

Comment:
Just my 2 cents, but I'd move this section to be another subsection of the previous section, alongside **Color**, **Variations**, and **Sizing** in the page hierarchy. I don't think we have a top-level **Localization** section in any of the other Design tabs (although I didn't do a full audit to check), and since this content is more informational than anything else, I feel like it would fit better alongside those other items.

Reply:
I think this suggestion makes sense.